diff --git a/htdocs/lib/pear/XML/Feed/Parser/RSS2.php b/htdocs/lib/pear/XML/Feed/Parser/RSS2.php
index c5d79d1..308a4ab 100644
--- a/htdocs/lib/pear/XML/Feed/Parser/RSS2.php
+++ b/htdocs/lib/pear/XML/Feed/Parser/RSS2.php
@@ -321,7 +321,8 @@ class XML_Feed_Parser_RSS2 extends XML_Feed_Parser_Type
      */
     function getLink($offset, $attribute = 'href', $params = array())
     {
-        $links = $this->model->getElementsByTagName('link');
+        $xPath = new DOMXPath($this->model);
+        $links = $xPath->query('//link');
 
         if ($links->length <= $offset) {
             return false;
